### ISSUE TITLE
Can't start due to `INFO: Waiting for changelog lock....`

### ISSUE THREAD

**Issue by charlietomo** [hassio-addons/addon-traccar#17]
# Problem/Motivation

> Can't get the addon to run; has an error in log. Does not look like a platform specific error (aarch64)

## Expected behavior

> Addon should run

## Actual behavior

> Install addon, tweak config, press start. Look at the log:

```
[22:02:38] INFO: Starting Traccar server...
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/opt/traccar/lib/slf4j-jdk14-1.7.26.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/opt/traccar/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.JDK14LoggerFactory]
2019-07-14 22:02:41  INFO: HikariPool-1 - Starting...
2019-07-14 22:02:42  INFO: HikariPool-1 - Start completed.
2019-07-14 22:02:45  INFO: Clearing database change log checksums
2019-07-14 22:02:45  INFO: SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK
2019-07-14 22:02:45  INFO: SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK
2019-07-14 22:02:45  INFO: SELECT LOCKED FROM PUBLIC.DATABASECHANGELOGLOCK WHERE ID=1
2019-07-14 22:02:45  INFO: Waiting for changelog lock....
2019-07-14 22:02:55  INFO: SELECT LOCKED FROM PUBLIC.DATABASECHANGELOGLOCK WHERE ID=1
2019-07-14 22:02:55  INFO: Waiting for changelog lock....
2019-07-14 22:03:05  INFO: SELECT LOCKED FROM PUBLIC.DATABASECHANGELOGLOCK WHERE ID=1
2019-07-14 22:03:05  INFO: Waiting for changelog lock....
2019-07-14 22:03:15  INFO: SELECT LOCKED FROM PUBLIC.DATABASECHANGELOGLOCK WHERE ID=1
2019-07-14 22:03:15  INFO: Waiting for changelog lock....
2019-07-14 22:03:25  INFO: SELECT LOCKED FROM PUBLIC.DATABASECHANGELOGLOCK WHERE ID=1
2019-07-14 22:03:25  INFO: Waiting for changelog lock....
```

With the duplicated lines just continuing to duplicate.

## Steps to reproduce

> Install, configure, press start.

## Proposed changes

> I don't know how to fix this; expect perhaps setting up with MySQL.

**Comment by frenck** [comment]
Duplicate of #15 and #5.

TL;DR: H2 is not useable for production, use MySQL as per add-on documentation.

**Comment by charlietomo** [comment]
I suggest you update the documentation if H2 doesn't actually work at all. I am wanting to do some initial tests (not use this in production) and I had read the documentation, which suggests this should work fine.

> By default, Traccar uses a database that is called H2. It is sweet, since it works out of the box. However, it is not considered stable for production use, so only use it for evaluation and testing purposes.

@frenck thanks for all the work you do with these add-ons.
